test.ts

```diff
@@ -31,6 +56,7 @@ macro('transpiles ES2020+ syntax', 'syntax.js')
 macro('transpiles ES2020+ syntax', 'syntax.js', {targets: 'last 2 Chrome versions'})
 macro('transpiles ES2020+ syntax', 'syntax.js', {loose: false})
 macro('transpiles ES2020+ syntax', 'syntax.js', {runtime: false})
+macro('transpiles ES2020+ syntax', 'syntax.js', {loose: false, runtime: false})
```

Comment:
Sorry, I missed your mention of this in the PR body. I understand now that this is so that some of the modern env snapshots will use Babel helpers. I have two concerns with this approach, though. One is that it's indirect: unless you've read that note in the PR body, it's not obvious what this combination of options is supposed to test. The other is that it's brittle: Babel could decide to stop using helpers to transpile this particular combination of options and syntax at any time; nothing about this config ensures that helpers will be used.

I think we should remove this test from this PR. If you think it's important to test this, could you make a follow-up issue to look at it separately?

Reply:
The modern environments had the same output from syntax.js for both `{runtime: true}` and `{runtime: false}`, so this was a way to force babel to include some helper functions that could be inlined, via the `loose` param forcing the `defineProperty` helper to be used.

Reply:
I'm not sure I understand the reasoning for the second point. If babel decides to compile things differently, all of our snapshots will change anyways, and we will detect it. And that kind of argument can also be made for every feature. If babel changes the way it compiles, every feature test potentially becomes obsolete. At this moment it is providing a valid check, so I don't see why we should be so afraid of future changes, which are not likely to happen anyway.

In response to the first point, we already have tests which exercise multiple parameter options with `{emotion: true, react: {runtime: 'classic'}}`. In that case you would also need context about the compilation details to understand why it's necessary. Is there a reason to treat that case differently from this one?